plugins/chat/spec/system/list_channels/mobile_spec.rb

```diff
@@ -201,6 +201,60 @@
     end
   end
 
+  context "when chat_preferred_mobile_index is set to direct_messages" do
+    before { SiteSetting.chat_preferred_mobile_index = "direct_messages" }
+
+    it "changes the default index" do
+      visit("/chat")
+
+      expect(page).to have_current_path("/chat/direct-messages")
+    end
+
+    context "when user can't use direct messages" do
+      before { SiteSetting.direct_message_enabled_groups = Group::AUTO_GROUPS[:staff] }
```

Comment:
I think this might be a false positive, it looks like we need to set the site setting otherwise the group won't be checked based on the router logic